rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml

```diff
@@ -0,0 +1,51 @@
+action: global
+title: CVE-2021-31979 CVE-2021-33771 Exploits by Sourgum
+id: ad7085ac-92e4-4b76-8ce2-276d2c0e68ef
+status: experimental
+description: Detects patterns as noticed in exploitation of Windows CVE-2021-31979 CVE-2021-33771 vulnerability and DevilsTongue malware by threat group Sourgum
+author: Sittikorn S
+date: 2021/07/16
+references:
+    - https://www.microsoft.com/security/blog/2021/07/15/protecting-customers-from-a-private-sector-offensive-actor-using-0-day-exploits-and-devilstongue-malware/
+    - https://citizenlab.ca/2021/07/hooking-candiru-another-mercenary-spyware-vendor-comes-into-focus/
+tags:
+    - attack.credential_access
+    - attack.t1566
+    - attack.t1203
+    - cve.2021-33771
+    - cve.2021-31979
+    - threat_group.Sourgum
+falsepositives:
+    - Unlikely
+level: critical
+---
+logsource:
+    product: windows
+    category: file_event
+detection:
+    selection:
+        TargetFileName|contains:
+            - 'C:\Windows\system32\physmem.sys'
+            - 'C:\Windows\System32\IME\IMEJP\imjpueact.dll'
+            - 'C:\Windows\system32\ime\IMETC\IMTCPROT.DLL'
+            - 'C:\Windows\system32\ime\SHARED\imecpmeid.dll'
+            - 'C:\Windows\system32\config\spp\ServiceState\Recovery\pac.dat'
+            - 'C:\Windows\system32\config\cy-GB\Setup\SKB\InputMethod\TupTask.dat'
+            - 'C:\Windows\system32\config\config\startwus.dat'
+            - 'C:\Windows\system32\ime\SHARED\WimBootConfigurations.ini'
+            - 'C:\Windows\system32\ime\IMEJP\WimBootConfigurations.ini'
+            - 'C:\Windows\system32\ime\IMETC\WimBootConfigurations.ini'
+    condition: selection
+---
+logsource:
+    product: windows
+    service: sysmon
```

Comment:
Please remove `service` as it is not need.
When you convert `registry_event` and `windows` it give sysmon chanel and the good eventid in the query.
Thanks

Reply:
Done